src/scripts/main.js

```diff
@@ -1,3 +1,380 @@
 'use strict';
 
-// write code here
+const header = document.querySelector('thead');
+const body = document.querySelector('tbody');
+const notifTimeout = 3000;
+const lengthName = 4;
+const ageMin = 18;
+const ageMax = 90;
+const employeeForm = document.createElement('form');
+
+employeeForm.classList.add('new-employee-form');
+
+employeeForm.innerHTML = `
+    <label>Name:
+      <input type="text" name="name" data-qa="name">
+    </label>
+    <label>Position:
+      <input type="text" name="position" data-qa="position">
+    </label>
+    <label>Office:
+      <select name="office" data-qa="office">
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      </select>
+    </label>
+    <label>Age:
+      <input type="number" name="age" data-qa="age">
+    </label>
+    <label>Salary:
+      <input type="number" name="salary" data-qa="salary">
+    </label>
+    <button type="submit">Save to table</button>
+`;
+
+document.body.append(employeeForm);
+
+let cIndex = -1;
+
+header.addEventListener('click', (e) => {
+  const target = e.target;
+  const index = target.cellIndex;
+
+  sortingTable(index, target.innerHTML, cIndex === index);
+
+  cIndex = (cIndex === index) ? -1 : index;
+});
+
+function sortingTable(index, nameColumn, directSorting) {
+  const sortTable = [...body.children];
+
+  sortTable.sort((first, second) => {
+    const a = first.children[index].innerHTML;
+    const b = second.children[index].innerHTML;
+
+    switch (nameColumn) {
+      case 'Name':
+      case 'Position':
+      case 'Office':
+        return a.localeCompare(b);
+
+      case 'Age':
+      case 'Salary':
+        return toNumber(a) - toNumber(b);
+    }
+  });
+
+  if (directSorting) {
+    sortTable.reverse();
+  }
+
+  body.append(...sortTable);
+};
+
+function toNumber(string) {
+  let res = '';
+
+  string.includes('$')
+    ? res = string.slice(1).split(',').join('')
+    : res = string;
```

Comment:
ternary operator shoul return value
```suggestion
  const res = string.includes('$')
    ? string.slice(1).split(',').join('')
    : string;
```